.github/workflows/run-ci.yaml

```diff
@@ -43,25 +43,25 @@ jobs:
       - name: Wait for the mock servers to be healthy
         run: |
           cd sinch-sdk-mockserver
-          while ! docker inspect --format='{{json .State.Health.Status}}' $(docker-compose ps -q authentication-server) | grep -q '"healthy"'; do
-            echo "Waiting for authentication-server to be healthy..."
-            sleep 2
-          done
-          while ! docker inspect --format='{{json .State.Health.Status}}' $(docker-compose ps -q fax-server) | grep -q '"healthy"'; do
-            echo "Waiting for fax-server to be healthy..."
-            sleep 2
-          done
-          while ! docker inspect --format='{{json .State.Health.Status}}' $(docker-compose ps -q numbers-server) | grep -q '"healthy"'; do
-            echo "Waiting for numbers-server to be healthy..."
-            sleep 2
-          done
-          while ! docker inspect --format='{{json .State.Health.Status}}' $(docker-compose ps -q conversation-server) | grep -q '"healthy"'; do
-            echo "Waiting for conversation-server to be healthy..."
-            sleep 2
-          done
-          while ! docker inspect --format='{{json .State.Health.Status}}' $(docker-compose ps -q conversation-templates-server) | grep -q '"healthy"'; do
-            echo "Waiting for conversation-templates-server to be healthy..."
-            sleep 2
+          servers=(
```

Comment:
💡 By creating a dedicated script onto sinch-mock-server, it:
- could be called by CI
- could be shared across SDKs
- will enable script usage to setting up env locally outside of CI

Reply:
Ticket created: https://tickets.sinch.com/browse/DEVEXP-493